docs/pages/management/dynamic-resources/teleport-operator-helm.mdx

```diff
@@ -0,0 +1,182 @@
+# CoreDNS is running at https://127.0.0.1:6443/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+# Metrics-server is running at https://127.0.0.1:6443/api/v1/namespaces/kube-system/services/https:metrics-server:https/proxy
+```
+
+<Admonition type="tip">
+  Users wanting to experiment locally with the operator can use [minikube](https://minikube.sigs.k8s.io/docs/start/)
+  to start a local Kubernetes cluster:
+  ```code
+  $ minikube start
+  ```
+</Admonition>
+
+## Step 1/3. Install teleport-cluster Helm chart with the operator
+
+(!docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx!)
+
+Install the Helm chart for the Teleport Cluster with `operator.enabled=true`:
+
+<Tabs>
+<TabItem scope="oss" label="Teleport Community Edition">
+
+```code
+$ helm install teleport-cluster teleport/teleport-cluster \
+        --create-namespace --namespace teleport-cluster \
+        --set clusterName=teleport-cluster.teleport-cluster.svc.cluster.local \
+        --set operator.enabled=true \
+        --version (=teleport.version=)
+```
+</TabItem>
+<TabItem scope="enterprise" label="Teleport Enterprise">
+
+Create a namespace for your Teleport cluster resources:
+
+```code
+$ kubectl create namespace teleport-cluster
+```
+
+(!docs/pages/includes//enterprise/obtainlicense.mdx!)
+
+Create a secret called "license" in the namespace you created:
+
+```code
+$ kubectl -n teleport-cluster create secret generic license --from-file=license.pem
+```
+
+Deploy your Teleport cluster and the Teleport Kubernetes Operator:
+
+```code
+$ helm install teleport-cluster teleport/teleport-cluster \
+        --namespace teleport-cluster \
+        --set enterprise=true \
+        --set clusterName=teleport-cluster.teleport-cluster.svc.cluster.local \
+        --set operator.enabled=true \
+        --version (=teleport.version=)
+```
+</TabItem>
+
+</Tabs>
+
+This command installs the required Kubernetes CRDs and deploys the Teleport Kubernetes Operator next to the Teleport
+cluster. All resources (except CRDs, which are cluster-scoped) are created in the `teleport-cluster` namespace.
+
+All subsequent commands will take place in the `teleport-cluster` namespace, so let's use it by default:
+```code
+$ kubectl config set-context --current --namespace teleport-cluster
+```
+
+## Step 2/3. Manage Teleport users and roles using `kubectl`
+
+Create a manifest called `teleport-resources.yaml` that describes two custom resources: a `TeleportUser` and a `TeleportRole`:
+
+```yaml
+apiVersion: resources.teleport.dev/v5
+kind: TeleportRole
+metadata:
+  name: myrole
+spec:
+  allow:
+    rules:
+      - resources: ['user', 'role']
+        verbs: ['list','create','read','update','delete']
+---
+apiVersion: resources.teleport.dev/v2
+kind: TeleportUser
+metadata:
+  name: myuser
+spec:
+  roles: ['myrole']
+```
+
+Apply the manifests to the Kubernetes cluster:
+
+```code
+$ kubectl apply -f teleport-resources.yaml
```

Comment:
force the namespace since only resources in `teleport-cluster` namespace will be watched.
If users follow this guide they won't have the resources synchronized

Reply:
The namespace is already set at the end of step 1:
> kubectl config set-context --current --namespace teleport-cluster

I also added a troubleshooting step.

Reply:
Can we specify the namespace instead of overriding the default one?
When you later visit the guide you won't jump to the beginning of it to configure the namespace.

Also, overriding users default config isn't a good process because sometimes users use kubeconfigs outside of $HOME/.kube/config.